Source/MgsLib/LibDG.cpp

```diff
@@ -645,12 +645,56 @@ MGS_FUNC_IMPLEX(0x401A4F, LibGvInitDispEnv_401A4F, LIBDG_IMPL);
 const char k0132_byte_650174[4] = { 0, 1, 3, 2 };
 
 void CC GV_kmd_zmd_file_handler_helper_4028C1(kmdObject* pKmdObj, kmdObject* pParentObj);
-MGS_FUNC_IMPLEX(0x4028C1, GV_kmd_zmd_file_handler_helper_4028C1, false); // TODO
+MGS_FUNC_IMPLEX(0x4028C1, GV_kmd_zmd_file_handler_helper_4028C1, LIBDG_IMPL); // TODO

```

Comment:
MGS_FUNC_IMPLEX can probably be moved after the function now that it is implemented and the prototype/TODO removed.